setup.py

```diff
@@ -16,6 +16,6 @@ def readme():
     author_email="support@nuvolos.cloud",
     license="MIT",
     packages=["nuvolos"],
-    install_requires=["pyodbc"],
+    install_requires=["pyodbc", "keyring"],

```

Comment:
getpass is missing?

Reply:
Also, please respect lexicogrpahic ordering

Reply:
getpass is from python itself, so we don't need to add it, right?

Reply:
ah ok, thx